services/frontend-service/src/ui/utils/Links.tsx

```diff
@@ -156,9 +164,17 @@ export const DisplayManifestLink: React.FC<{ displayString: string; app: string;
     props
 ): JSX.Element | null => {
     const { displayString, app, version } = props;
+    const frontendConfig = useFrontendConfig((m) => m);
+
+    let manifestLink = '/ui/manifest?app=' + app + '&release=' + version;
     const manifestRepo = useManifestRepoUrl();
     const branch = useBranch();
-    const manifestLink = deriveReleaseDirLink(manifestRepo, branch, app, version);
+
+    if (frontendConfig.configs.manifestRepoUrl !== '') {
```

Comment:
I think `manifestRepo` is the same as `frontendConfig.configs.manifestRepoUrl` => please simplify the code